src/app/TideSearchApplication.cpp

```diff
@@ -1783,8 +1783,7 @@ int TideSearchApplication::calcScoreCount(
         if (col == colLast) { 
```

Comment:
One thing that I think you missed that I didn't make clear in the last comment. I also suggested switching the order between the if and else if statement.

I believe "if (col <colLast)" would be faster than "if (coll==colLast)" as we expect the first one to be true more often.